definitions/npm/axios_v0.19.x/flow_v0.80.x-/axios_v0.19.x.js

```diff
@@ -4,16 +4,16 @@ declare module 'axios' {
 
   declare type AxiosTransformer<T> = (
     data: T,
-    headers?: { [key: string]: any }
-  ) => any;
+    headers?: {| [key: string]: mixed |},
```

Comment:
I don't think [an exact object works](https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVAXAngBwKZgDyAdgQLxgDeAPmANoDWeWAXGAM4YBOAlsQOYBdNgFseADzwATMNQC+AblRgVqtStQw8GMBjyc2JclTBQ4cNgHIzcS2EVhUQA) here.

Later in the diff you used `{ [key: string]: mixed, ...}` for such case, any special thing here?

Reply:
Good call, I think the original thought was flow will match up the type instead of literal key string, will this fixed and test covered

Reply:
Maybe I misunderstood your comment, @quanlinc, but I'd like to clarify: We don't want to enumerate the possible keys here (they can be anything). I think we just need to make this inexact instead.

Reply:
It could be just me not expressing myself clearly, I agree with what @pascalduez said, I had a wrong interpretation of using exact type, inexact is the way to go.